test/test_foreach.py

```diff
@@ -132,6 +133,7 @@ def transform(t):
 
 # note(crcrpar): `zero_size` is `False` unless (dtype, device) == (torch.float32, "cuda")
 # as the pair would go through `multi_tensor_apply_kernel` if inputs are not zero size.
+@unittest.mock.patch.dict(os.environ, {"KINETO_LOG_LEVEL": "5"})
```

Comment:
For my understanding, do you have docs/know what the levels correspond to? / How do you know to use 5 vs any other number?